### DISCUSSION TITLE
Slovak language

### DISCUSSION THREAD

**Discussion by flyer05** [KlipperScreen/KlipperScreen#1507] [Q&A] [answered]
How to add Slovak {sk} language localization/translation in Weblate?

**Comment by alfrix** [comment]
due to various requests that ended up with an empty language (that made selecting the language just show everything in English which looks like a bug), i disabled the add language feature.
to add a language you should translate the .pot file and post it here or in an Feature Request.
it's located at `/ks_includes/locales/KlipperScreen.pot` you can grab it from the repository.
i recommend that using poedit.

**Comment by alfrix** [comment]
in the future i'll probably add something that auto-disables languages if they don't meet the threshold of translation, so i can re-enable the weblate request and everything is a bit more automated, but for now you need to provide a translation file already done.

**Comment by flyer05** [comment]
Hello, I translated the pot file into Slovak and also compiled it into mo.
How is possible upload it, looks that I don't have permissions.

**Reply by alfrix** [reply]
> Hello, I translated the pot file into Slovak and also compiled it into mo. How is possible upload it, looks that I don't have permissions.

Hello, you need to commit it into your own repo
then do a pull request (via website is probably the easiest)

**Comment by alfrix** [comment, marked as answer]
finished with #1607
